src/libs/SidebarUtils.ts

```diff
@@ -298,6 +299,10 @@ function getOptionData(
     preferredLocale: ValueOf<typeof CONST.LOCALES>,
     policy: Policy,
     parentReportAction: ReportAction,
+    // TODO: Clarify typing
+    nextStep: {
+        requiresUserAction: boolean;
+    },
```

Comment:
@mountiny, how should I type `nextStep` for now?

Reply:
For now just generic object

Reply:
Done - 9b16ac3.